scan/src/components/ProofTab.re

```diff
@@ -0,0 +1,60 @@
+let copyStringToClipboard: (string, string) => unit = [%bs.raw
+  {|
+function copyStringToClipboard (str,strMsg) {
+   var element = document.createElement('textarea');
```

Comment:
What's the difference between using this vs using https://www.npmjs.com/package/clipboard-copy, similar to what we did in staking portal. 

Can you confirm the solution you are doing will work on all platforms?